Makefile

```diff
@@ -54,6 +54,9 @@ init: node_modules ;
 # Use building English language as a check to see if everything works
 test: en ;
 
+lint:
```

Comment:
Good question. That makes a lot of sense to me although note that CI already runs it.